sql1.txt

```diff
@@ -6,38 +6,79 @@ SELECT * FROM Products LIMIT 5;
 1. Use an SQL SELECT statement to retrieve the first 10 rows of the Customer table, ordered by CustomerName.
 Paste your SQL statement below:
 
+SELECT * 
+FROM Customers 
+ORDER BY CustomerName LIMIT 10;
 
 2. Use an SQL Select statement to retrieve the names and prices for all products that cost less than $20.
 Paste your SQL statement below:
 
+SELECT ProductName, Price 
+FROM Products WHERE Price < 20.0;
 
 3. Retrieve all employees whose last name starts with C.  Paste your SQL statement below:
 
+SELECT * 
+FROM Employees 
+WHERE LastName LIKE  "C%";
 
 4. Retrieve the order ID and customer name for all orders where the customer name starts with A.
 Here you will have to do a join of the Orders table and the Customers table.  Paste your SQL
 statement below.
 
+SELECT o.OrderID, c.CustomerName 
+FROM Orders o 
+JOIN Customers c ON o.CustomerID = c.CustomerID 
+WHERE c.CustomerName LIKE 'A%';
 
 5. Retrieve the list of customers, ordered by customer name, along with their order IDs.  Note
 that not every customer has an order, but be sure that you include those customers without orders
 in the results of the query.  Paste your SQL statement below.
 
+SELECT c.CustomerName, o.OrderID 
+FROM Customers c 
+FULL JOIN Orders o ON o.CustomerID = c.CustomerID 
+ORDER BY c.CustomerName;
 
+or
+
+SELECT c.CustomerName, o.OrderID 
+FROM Customers c 
+LEFT JOIN Orders o ON o.CustomerID = c.CustomerID 
+ORDER BY c.CustomerName
 
 6. Retrieve the list of customer names for customers that have no orders.  There are two ways to do this
 One is to use a subquery, as follows:
 
-SELECT CustomerName FROM Customers WHERE CustomerID NOT IN (SELECT CustomerID FROM ORDERS);
-
+SELECT * F
+ROM Customers 
+WHERE NOT EXISTS (
+	SELECT 1 
+	FROM Orders 
+	WHERE Orders.CustomerID = Customers.CustomerID
+);
 This is a subquery -- we haven't talked about those.  But, you can do the same thing with a left
 join by comparing the OrderID column with NULL.  Paste your SQL statement below.
 
+SELECT c.CustomerName, o.OrderID 
+FROM Customers c 
+FULL JOIN Orders o ON o.CustomerID = c.CustomerID 
+WHERE o.OrderID IS NULL;
+
+Or 
 
+SELECT c.CustomerName, o.OrderID 
+FROM Customers c 
+FULL JOIN Orders o ON o.CustomerID = c.CustomerID 
+WHERE o.OrderID IS NULL;
```

Comment:
This is the same query as the one on lines 63-66. You might have meant to say `LEFT JOIN`?

Reply:
> This is the same query as the one on lines 63-66. You might have meant to say `LEFT JOIN`?

Right, my bad. Thank you! Appreciate your feedback!